Gemfile

```diff
@@ -80,7 +80,7 @@ group :jekyll_optional_dependencies do
     gem "pygments.rb", "~> 1.0"
     gem "rdiscount", "~> 2.0"
     gem "redcarpet", "~> 3.2", ">= 3.2.3"
-    gem "yajl-ruby", "~> 1.3.1"
+    gem "yajl-ruby", "~> 1.4.0"
```

Comment:
Any reason not to use `"~> 1.3"` ?

Reply:
Just updated as mentioned in #6965

Reply:
Using `"~> 1.3"` is better..

Reply:
Ah, got it, sorry :) Read '1.3.1' instead of '1.3'